src/cursor/abstract_cursor.ts

```diff
@@ -43,7 +43,9 @@ export const CURSOR_FLAGS = [
   'partial'
 ] as const;
 
-/** @public */
+/** @public
+ *  @deprecated Not used
+ * */
 export interface CursorCloseOptions {
   /** Bypass calling killCursors when closing the cursor. */
   skipKillCursors?: boolean;
```

Comment:
I think we should deprecate just this option, I know the whole thing is unused but if we were to add options later then we wouldn't be un-deprecating but lets see what the team says.

Reply:
I agree with deprecating just the one option, which seemed to be the intent of the ticket anyway; I think if we deprecate the entire options object, it would make sense to also remove some of the overloads that use the options, which would be out of scope here, and would make it more work to add options in the future; my personal opinion is that if we manage to go an entire version without adding any options then we can consider removing the interface entirely for the next major release

Reply:
@W-A-James I think we still want the deprecation on the skipKillCursors as well as the overloads

Reply:
@nbbeeken Ah I see what you and Daria mean now. In that case, should I go ahead and remove the deprecation from those function signatures?